src/tosam.py

```diff
@@ -1,18 +1,21 @@
-import argparse
-
-
-def main():
-    argparser = argparse.ArgumentParser(description="To Simple-SAM converter")
-    argparser.add_argument(
-        "mas",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
-    print(args)
-    for line in args.mas:
-        chrom, read_name, read_str, pos = line.split('\t')
-        # Output as Simple-SAM
-
-
-if __name__ == '__main__':
-    main()
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(prog = "tosam")
+parser.add_argument("-path", nargs = 1, type = str)
+
+args = parser.parse_args()
+
+file_object = open(r"{path}".format(path=args.path[0]), "r")

```

Comment:
I believe `args.path[0]` is already a string, so `r"{path}".format(path=args.path[0])` is just a complicated and slow way of writing `args.path[0]`. Should you wish to format strings, however, `f"{args.path[0]}"` would do the trick; it is a few versions of Python ago that `"...".format(...)` was the recommended practice.

Of course, if you had just let `argparse` deal with file arguments you wouldn't need this in the first place.